app/proto/requests.proto

```diff
@@ -90,47 +78,5 @@ message ListHostRequestsReq {
 
 message ListHostRequestsRes {
   repeated HostRequest host_requests = 1;
-  int64 last_request_id = 2;
-  bool no_more = 3;
-}
-
-message GetHostRequestMessagesReq {
-  int64 host_request_id = 1;
-  int64 last_message_id = 2;
-  uint32 number = 3;
-}
-
-message GetHostRequestMessagesRes {
-  repeated org.couchers.api.conversations.Message messages = 1;
-  int64 last_message_id = 4;
-  bool no_more = 5;
-}
-
-message SendHostRequestMessageReq {
-  int64 host_request_id = 1;
-  string text = 2;
-}
-
-message GetHostRequestUpdatesReq {
-  int64 newest_message_id = 1;
-  uint32 number = 2;
-  bool only_sent = 3;
-  bool only_received = 4;
-}
-
-message HostRequestUpdate {
-  int64 host_request_id = 1;
-  org.couchers.api.conversations.Message message = 2;
-  // the current status of the host request, regardless of this update
-  org.couchers.api.conversations.HostRequestStatus status = 3;
-}
-
-message GetHostRequestUpdatesRes {
-  repeated HostRequestUpdate updates = 1;
-  bool no_more = 2;
-}
-
-message MarkLastSeenHostRequestReq {
-  int64 host_request_id = 1;
-  int64 last_seen_message_id = 2;
```

Comment:
I assume this was all deleted because the front end doesn't use it? Or was this just wrapped up into the general message stuff? In either event, do we need to remove backend functions and tests?